doc/api/assert.md

```diff
@@ -143,6 +151,9 @@ changes:
     pr-url: https://github.com/nodejs/node/pull/3276
     description: The `error` parameter can now be an arrow function.
 -->
+* `block` {Function}
+* `error` {Error|RegExp|Function}
```

Comment:
The documentation doesn't mention how anything other than a `RegExp` or a `Function` (either a constructor or a validation function) would work. I.e., I don't think it can be an `Error` object, only an `Error` constructor. Ditto for `.throws()`.